### PR TITLE
navigation menu on both screen sizes.

### PR DESCRIPTION
**Pull request by sergiogval**
This feature includes:

- [x] Menu mobile functionality.
- [x] Responsivness to mobile and desktop version.
- [x] The firste versión of the landing page.

I have to point out that I'm making the all page from the scratch. Pure vanilla flavor.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        <p class="intro">
+          Hello! I'm an economist who also does software development!
+          I can help you build an business case,
+          product, feature or website. Take a look of my works.
+          Don’t hesitate and contact me.
+        </p>
+        <button class="btn-1" id="btn-1" type="button">Start collaboration</button>
+      </div>
+      <div>
+        <img id="desk-image" class="desk-image" src="images/landingdesk.png" alt="landingdesk" />
```
Comment (sergiogval) [inline index.html:43]: This element needs to be fixed. It overflows the all page.

File: js/topnav.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+const menu = document.querySelector('.menu');
+const close = document.querySelectorAll('.ham span');
+// const mobileImage = document.querySelector('.background-image');
+const links = document.querySelectorAll('.link-sections');
+
+// Mobile Menu Function
+ham.addEventListener('click', () => {
+  menu.classList.toggle('active');
+  navbar.classList.toggle('active');
+  // mobileImage.style.backgroundImage = "url('images/mobile-menu-img.png')";
```
Comment (sergiogval) [inline js/topnav.js:13]: This image need to be shown when the mobile menu is open

File: css/topnav.css [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+    20%,
+    20%;
+  background-position-x:
+    0%,
+    12%;
+  z-index: 1;
+  transition: 0.2s 0.1s;
+}
+
+.br-1.animated {
```
Comment (sergiogval) [inline css/topnav.css:156]: Fix this selectors in order to make a cross when the mobile menu is open

**Review by Rustamxon7 — APPROVED** [review]
## Hi @sergiogval   👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**